app/customer.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass
+from app.shop import Shop
+from typing import List
+
+
+@dataclass
+class Customer:
+    argus: dict
+    name: str = None
+    product_cart: dict = None
+    location: list = None
+    money: int | float = None
+    car: dict = None
+    cheapest_shop: Shop | None = None
```

Comment:
Do you really want to be able to create Customer with such None values? Remove that